content/applications/hr/payroll/contracts.rst

```diff
@@ -12,214 +12,351 @@ about their position.
-
-Last, enter the :guilabel:`Amount` that each payslip pays towards the entry.
+.. important::
+   When the total amount has been paid for the salary attachment, navigate to the individual salary
+   attachment and click the :guilabel:`Mark as Completed` button at the top of the form. This
+   changes the status to :guilabel:`Completed` and the garnishments are no longer taken out of the
+   employee's paychecks.
 
-To delete a line, click the :guilabel:`🗑️ (trash can)` icon at the end of the line.
+Offers
+======
 
-Save and send the contract
---------------------------
+Once a contract has been created and/or modified, the contract must be sent to the employee to be
+accepted and signed.
 
-Once a contract has been created and/or modified, save the contract by clicking the :guilabel:`Save`
-button. Next, the contract must be sent to the employee to be signed.
+Send an offer
+-------------
 
-Click on one of the following buttons to send the contract to the employee:
+Open an individual contract by navigating to :menuselection:`Payroll app --> Contracts ->
+Contracts`, and click on a contract to open the contract form. Click on the :guilabel:`Generate
+Offer` button at the top of the page and a :guilabel:`Generate Simulation Link` pop-up appears.
 
 .. image:: contracts/send-contract.png
    :align: center
    :alt: Send the contract to the employee via one of the buttons.
 
-- :guilabel:`Generate Simulation Link`: This option is for Belgian companies only. Clicking this
-  opens a pop-up window that contains the basic information from the contract as well as a link for
-  the contract when using the salary configurator. Click :guilabel:`Send Offer` to send an email to
-  the employee so they can sign the contract.
+The :guilabel:`Generate Simulation Link` pop-up form contains all the information pulled from the
+contract, including the :guilabel:`Contract Template`, :guilabel:`Job Position`, :guilabel:`Job
+Title`, :guilabel:`Department`, :guilabel:`Contract Start Date`, :guilabel:`Default Vehicle`,
+:guilabel:`Contract Type`, and the :guilabel:`Yearly Cost`.
 
-  .. image:: contracts/simulation.png
-     :align: center
-     :alt: Sends a link to the employee for the contract.
+At the bottom of the pop-up is a :guilabel:`Link Expiration Date`. This is the timeframe that the
+contract offer is valid for. The default that is pre-populated is `30 days`, but can be modified.
+
+Click the :guilabel:`Send By Email` button and a different pop-up appears, using the send offer
+email template. Make any modifications to the email, and attach any additional documents needed,
+then click :guilabel:`Send` to send the offer.
+
+.. image:: contracts/send-offer.png
+   :align: center
+   :alt: The email template pop-up to send an offer.
 
   .. note::
      In order to send a contract using the :guilabel:`Generate Simulation Link`, there must be a
      signature field in the contract PDF being sent to the employee so they can sign it.
```

Comment:
Neither the **image** or the **note** are rendering here because of the improper indentation of the **note**. we can't embed a **note** _inside_ of an **image** directive.

Here's the fix which will resolve the failing CI check

```suggestion
.. image:: contracts/send-offer.png
   :align: center
   :alt: The email template pop-up to send an offer.

.. note::
   In order to send a contract using the :guilabel:`Generate Simulation Link`, there must be a
   signature field in the contract PDF being sent to the employee so they can sign it.
```

Reply:
Thank you! I kept looking and looking and did not understand what was wrong with it.  I didn't realize I had an extra space- I was hung up on the way the image part was entered!